prototypes/recordedfuture.yml

```diff
@@ -35,7 +31,7 @@ prototypes:
                 share_level: red
 
     DomainRiskList:
-        author: Jonas Eichinger
+        author: Jonas Eichinger, Recorded Future Inc.
```

Comment:
Same here. I would not change the original author of this prototype, but I would mark this as deprecated. The new Master Miner can do more than the original miner.

Reply:
I will change the name back to its original name. For marking it as deprecated, Should I write something like this :

prototypes:
IPRiskList:
author: Jonas Eichinger (deprecated)

We are planning to keep this miner active for our clients.

Reply:
you can change `development_status` to `DEPRECATED` and leave the author as it is